recipe/meta.yaml

```diff
@@ -128,6 +130,7 @@ outputs:
       strong:
         - {{ pin_subpackage("libgfortran-ng") }}
         - {{ pin_subpackage("libgcc-ng", max_pin=None) }}
+        - {{ pin_subpackage("openmp_impl", max_pin=None) }}
```

Comment:
I don't know if this preferable or not. cc @mingwandroid.
Also, what do you think about renaming `openmp_impl` -> `_openmp_mutex` like the other mutexes?

Reply:
To expand on the first point, do we add this to all packages even those that don't need openmp or do we want to require package authors to explicitly declare an openmp dependency?

Reply:
I am fine renaming the mutex. I used the name you suggested initially in your comments on the openmp megathread.

We have to have a run export on openmp_impl at least the compiler activation scripts. It contains libgomp.so.1 and so conda-build is erroring because we are forcing it to be more careful about under/overlinking.

Reply:
run exports are not transitive and so it is not making it in even though it is a proper dep of libgcc-ng

Reply:
To address your actual question, right now for linux we ship openmp without requiring anything extra. We can switch to requiring it I suppose, but this will break a lot of downstream builds.